<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdvertDialog</class>
 <widget class="QDialog" name="AdvertDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>515</width>
    <height>585</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>537</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>50</width>
    <height>50</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Ajouter/Modifier une annonce</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>411</width>
     <height>581</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="vl_main">
    <item>
     <layout class="QFormLayout" name="fm_add">
      <item row="1" column="0">
       <widget class="QLabel" name="lbl_type">
        <property name="layoutDirection">
         <enum>Qt::RightToLeft</enum>
        </property>
        <property name="text">
         <string>Type de bien</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="cbx_type">
        <property name="editable">
         <bool>false</bool>
        </property>
        <item>
         <property name="text">
          <string>Maison</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Appartement</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Bureau</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Commerce</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Ferme</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Château</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="lbl_surface">
        <property name="layoutDirection">
         <enum>Qt::RightToLeft</enum>
        </property>
        <property name="text">
         <string>Surface / Superficie</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <layout class="QHBoxLayout" name="hl_m2">
        <item>
         <widget class="QLineEdit" name="le_size"/>
        </item>
        <item>
         <widget class="QLabel" name="lbl_m2">
          <property name="text">
           <string>m2</string>
          </property>
          <property name="textFormat">
           <enum>Qt::AutoText</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="lbl_nb_room">
        <property name="layoutDirection">
         <enum>Qt::RightToLeft</enum>
        </property>
        <property name="text">
         <string>Nombres de pièces</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QSpinBox" name="spb_rooms"/>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="lbl_address">
        <property name="layoutDirection">
         <enum>Qt::RightToLeft</enum>
        </property>
        <property name="text">
         <string>Adresse</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <layout class="QVBoxLayout" name="vl_address" stretch="0,0">
        <property name="spacing">
         <number>-1</number>
        </property>
        <property name="sizeConstraint">
         <enum>QLayout::SetDefaultConstraint</enum>
        </property>
        <item>
         <layout class="QHBoxLayout" name="hl_num_street">
          <item>
           <widget class="QLineEdit" name="le_num">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>1</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="placeholderText">
             <string>Num</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="le_street">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>5</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="placeholderText">
             <string>Rue</string>
            </property>
            <property name="clearButtonEnabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="hl_city_code">
          <item>
           <widget class="QLineEdit" name="le_city">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>3</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="placeholderText">
             <string>Ville</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="le_zip">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>1</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="placeholderText">
             <string>Code</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="lbl_description">
        <property name="layoutDirection">
         <enum>Qt::RightToLeft</enum>
        </property>
        <property name="text">
         <string>Description</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QTextEdit" name="te_description"/>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="lbl_price">
        <property name="layoutDirection">
         <enum>Qt::RightToLeft</enum>
        </property>
        <property name="text">
         <string>Prix</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QRadioButton" name="rdb_sale">
          <property name="text">
           <string>Vente</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rdb_rent">
          <property name="text">
           <string>Location</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="6" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLineEdit" name="le_price">
          <property name="maximumSize">
           <size>
            <width>150</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lbl_euro">
          <property name="text">
           <string>€</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="7" column="1">
       <layout class="QHBoxLayout" name="hl_photo_princ">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QToolButton" name="btn_photo_princ">
            <property name="text">
             <string>...</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="btn_suppr_princ">
            <property name="text">
             <string>Suppr</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="lbl_photo_princ">
        <property name="text">
         <string>Photo principale</string>
        </property>
       </widget>
      </item>
      <item row="8" column="1">
       <layout class="QHBoxLayout" name="hl_photo_sup">
        <item>
         <widget class="QToolButton" name="btn_photo_sup">
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="btn_suppr_sup">
          <property name="text">
           <string>Suppr</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="8" column="0">
       <widget class="QLabel" name="lbl_photo_sup">
        <property name="text">
         <string>Photo(s) supplé.</string>
        </property>
       </widget>
      </item>
      <item row="9" column="0">
       <widget class="QLabel" name="lbl_client">
        <property name="text">
         <string>Client</string>
        </property>
       </widget>
      </item>
      <item row="9" column="1">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLineEdit" name="le_client_name">
            <property name="placeholderText">
             <string>Nom</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="le_client_surname">
            <property name="placeholderText">
             <string>Prénom</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <property name="spacing">
           <number>1</number>
          </property>
          <item>
           <widget class="QLineEdit" name="le_client_tel">
            <property name="placeholderText">
             <string>Téléphone</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="lbl_etou">
            <property name="text">
             <string>et/ou</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="le_client_mail">
            <property name="placeholderText">
             <string>Email</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="hl_buttonBox">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btn_cancel">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>32</height>
         </size>
        </property>
        <property name="text">
         <string>Annuler</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_ok">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>32</height>
         </size>
        </property>
        <property name="focusPolicy">
         <enum>Qt::TabFocus</enum>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Ok</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
